openwhisk/openwhisk_catalog.md

```diff
@@ -609,6 +609,22 @@ For more information on the payload content see the [Github events and payload](
 
 The `/whisk.system/pushnotifications` package enables you to work with a push service. 
 
+### Install the IBM Push Notification OpenWhisk package 
```

Comment:
No need to add this section, in Bluemix the Push Notifications package comes installed by default. This is only required in open source openwhisk.